transactions/README.md

```diff
@@ -97,36 +107,69 @@ fileName:transaction.json
+</div>
 
-Refund transactions are processed when a purchased item is returned and the payment is reversed. Refund transactions are a type of clearing transactions that have negative `amount` and trigger the `transaction.clearing` webhook event.
 
-Fidel matches full refund transactions to its original transaction by comparing `cardId`, `locationId`, `merchantId`, `amount` and `datetime`. The refund `transactionId` is set in the original transaction as `refundTransactionId` and the original `transactionId` is set on the refund transaction as `originalTransactionId`.
+## Test Transactions
 
-## Test transactions
+For testing purposes, you can use the [**API Playground**](https://dashboard.fidel.uk/playground) in the Fidel Dashboard test environment to create test transactions and test your application logic. Alternatively, you can use the [Create Test Transaction](https://reference.fidel.uk/reference#create-transaction-test) API endpoint to create authorisation test transaction.
 
-For testing purposes, you can use the **API Playground** in test environment to create test transactions and test your application logic.
+To create a test transaction, you will need a [Program](/programs), a [Location](/locations) and a test [Card](cards) linked to the program.
 
-To create a test transaction you will need a program, a location and a test card linked to the program.  
+##### Create Test Transactions Using the API.
 
-On the dashboard, go to **API Playground** and click on **Create transaction** from the endpoints menu. The method will be set to POST and the endpoint to **_/transactions/test_**. An editable sample JSON object like the following one will be use to create the transaction.
+```sh
+curl -X POST \
+  https://api.fidel.uk/v1/transactions/test \
+  -H 'content-type: application/json' \
+  -H 'fidel-key: sk_test_50ea90b6-2a3b-4a56-814d-1bc592ba4d63' \
+  -d '{
+    "amount": 10,
+    "cardId": "bc538b71-31c5-4699-820a-6d4a08693314",
+    "locationId": "7a916fbd-70a0-462f-8dbc-bd7dbfbea140"
+  }'
+```
 
-To create a test transaction, use the dropdown menus to select the Program, Location and Card for the transaction.  These selections will be used to populate the `cardId`, `locationId` and the `amount` in the JSON payload.  You can modify any of the properties in the JSON file (including the amount). If the transaction is created successfully you will see the transaction object in the response body box.
+##### Create Test Transactions Using the API Playground.
 
-##### Create sample transactions in test mode using the API Playground.
+On the Fidel Dashboard, go to the [**Playground**](https://dashboard.fidel.uk/playground) option in the navigation menu. Click on **Create transaction** link, located on the left side, in the **END POINTS** menu. The method will be set to POST and the endpoint to **_/transactions/test_**. An editable sample JSON object like the following one will be used to create the transaction.
```

Comment:
`END POINTS` -> `ENDPOINTS` please. do we have to capitalise it?

Reply:
The Dashboard says "END POINTS", capitalized and with a space, so I'd like to keep it the same in the docs.

Reply:
uh oh, then it's wrong on the Dashboard, it will be fixed in the next version